pkg/Microsoft.ML.OnnxTransform/Microsoft.ML.OnnxTransform.nupkgproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk" DefaultTargets="Pack">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <PackageDescription>ML.NET component for Sonoma</PackageDescription>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="../Microsoft.ML/Microsoft.ML.nupkgproj" />
+    <PackageReference Include="Microsoft.ML.Scoring" Version="1.0.4-dev47509" />
```

Comment:
>1.0.4-dev47509 [](start = 62, length = 14)

please put it to [https://github.com/dotnet/machinelearning/blob/master/build/Dependencies.props](https://github.com/dotnet/machinelearning/blob/master/build/Dependencies.props)